README.md

```diff
@@ -18,7 +21,7 @@ This will **copy** the DLLs into the `system32` and `syswow64` directories of yo
 The setup script optionally takes the following arguments:
 - `--symlink`: Create symbolic links to the DLL files instead of copying them. This is especially useful for development.
 - `--with-d3d10`: Install the `d3d10{_1}.dll` helper libraries.
-- `--without-dxgi`: Do not install DXVK's DXGI implementation and use the one provided by wine instead. This is necessary for both vkd3d and DXVK to work within the same wine prefix.
```

Comment:
I mean, it's still required for vkd3d-non-proton :frog:.

Reply:
While true, I don't think much of our target audience is likely to care about upstream vkd3d at this point.

Reply:
Wdym? `vkd3d-proton` is a "waste of effort"! :frog: :D